src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.h

```diff
@@ -3,15 +3,20 @@
 
 #include "global.h"
 
+#define GET_KIBAKO2_COLLECTIBLE_ID(x) (((x->dyna.actor.params >> 0x8) & 0x7F) << 8)
+#define KIBAKO2_SKULLTULA_SPAWN_PARAM(x) (((x->dyna.actor.params & 0x1F) << 2) | 0xFF01)
```

Comment:
These are fine, but as noted, some more are needed.

Reply:
I think I have every macro/enum that makes sense outside of maybe this particularly gnarly looking thing in `ContainsSkulltula`
```c
 flag = ((actorSpawnParam & 0x3FC) >> 2) & 0xFF;
 ```
 
 But I think `Actor_Spawn` needs to be decompiled to really understand what this parameter means so I'd rather leave it for now.

Reply:
I agree, I'm sure there's other such monstrosities elsewhere that will come with better z_actor docs.